backend/account/views.py

```diff
@@ -74,7 +74,7 @@ def update_account(request: HttpRequest):
         # the User model (AttributeError), tries to modify a user that doesn't
         # exist (ObjectDoesNotExist), or doesn't supply the Google ID (KeyError)
         logger.debug(f"Update error:\n{err_msg}")
-        return JsonResponse(status=400, data={})
+        return JsonResponse(status=400, data={'error': repr(err_msg)})
```

Comment:
This might be a whole PR on its own to make things consistent - ive been using `HttpResponse(status=400)`- or could do as part of #74 to avoid conflicts